prisma/schema.prisma

```diff
@@ -127,6 +127,34 @@ model Claim {
   lastUpdatedAt DateTime @default(now())
 }
 
+// this is for validator service
+model CandidUserInfo {
+  id         Int    @id @default(autoincrement())
+  claimId    Int?   @unique
+  firstName  String
+  lastName   String
+  email      String
+  profileURL String
+}
+
+model Validation {
```

Comment:
maybe this is 'ValidationRequest'

the actual validation is a claim, in the "Claim" model